### PR TITLE
Emurasoft.EmEditor64 version 20.8.1

### PR DESCRIPTION
**Pull request by yutakaemura**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/17216)

### CODE REVIEW

File: manifests/e/Emurasoft/EmEditor64/20.8.1/Emurasoft.EmEditor64.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Emurasoft.EmEditor64
+PackageVersion: 20.8.1
+PackageLocale: en-US
+Publisher: Emurasoft
+PackageName: EmEditor (64-bit)
```
Comment (OfficialEsco) [inline manifests/e/Emurasoft/EmEditor64/20.8.1/Emurasoft.EmEditor64.locale.en-US.yaml:7]:
```suggestion
Publisher: Emurasoft, Inc.
PackageName: EmEditor
```
Needs to match ARP

File: manifests/e/Emurasoft/EmEditor64/20.8.1/Emurasoft.EmEditor64.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: Emurasoft.EmEditor64
```
Comment (OfficialEsco) [inline manifests/e/Emurasoft/EmEditor64/20.8.1/Emurasoft.EmEditor64.installer.yaml:3]:
```suggestion
PackageIdentifier: Emurasoft.EmEditor
```

File: manifests/e/Emurasoft/EmEditor64/20.8.1/Emurasoft.EmEditor64.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Emurasoft.EmEditor64
```
Comment (OfficialEsco) [inline manifests/e/Emurasoft/EmEditor64/20.8.1/Emurasoft.EmEditor64.locale.en-US.yaml:3]:
```suggestion
PackageIdentifier: Emurasoft.EmEditor
```

File: manifests/e/Emurasoft/EmEditor64/20.8.1/Emurasoft.EmEditor64.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: Emurasoft.EmEditor64
```
Comment (OfficialEsco) [inline manifests/e/Emurasoft/EmEditor64/20.8.1/Emurasoft.EmEditor64.yaml:3]:
```suggestion
PackageIdentifier: Emurasoft.EmEditor
```

File: manifests/e/Emurasoft/EmEditor64/20.8.1/Emurasoft.EmEditor64.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+PackageIdentifier: Emurasoft.EmEditor64
+PackageVersion: 20.8.1
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://emeditor.blob.core.windows.net/emed64_20.8.1.msi
+  InstallerSha256: 02344A0F4019BA0DE3A3D691EC0F9BCA1D52CDAD906BC6F314E359B7A98D7D0C
+  ProductCode: '{4CC1E959-6A31-4CF7-92E5-53ED64F68EA7}'
```
Comment (OfficialEsco) [inline manifests/e/Emurasoft/EmEditor64/20.8.1/Emurasoft.EmEditor64.installer.yaml:11]:
```suggestion
  ProductCode: '{4CC1E959-6A31-4CF7-92E5-53ED64F68EA7}'
- InstallerLocale: en-US
  Architecture: x86
  InstallerType: msi
  InstallerUrl: https://emeditor.blob.core.windows.net/emed32_20.8.1.msi
  InstallerSha256: 08CD3BC73AA6E6352394061B2E9BA01A5724B0EFD695036EDA87DF3C4CB70D5E
  ProductCode: '{E15BDCD0-F6EB-4416-8F9C-32BA1A0CC141}'
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Why are you splitting up the installer when it can be in the same manifest?..

**Review by yutakaemura — COMMENTED** [review]
I approve changes.

**Pull request closed without merging**